src/app/workbasket/filters/workbasket-filters.component.ts

```diff
@@ -145,15 +145,16 @@ export class WorkbasketFiltersComponent implements OnInit {
             .sort(this.orderService.sortAsc);
           const formValue = this.windowService.getLocalStorage(FORM_GROUP_VAL_LOC_STORAGE);
 
-          for (let i = 0; i < workbasketInputs.length; i++) {
-            let item = workbasketInputs[i];
+          workbasketInputs.forEach(item => {
+            if (item.field.elementPath) {
+              item.field.id = item.field.id + '.' + item.field.elementPath;
```

Comment:
can't see elementPath added to the Field model in this PR

Reply:
Field model is coming from the ui-toolkit. It will appear when ui-toolkit will get merged.